src/py42/clients/alerts.py

```diff
@@ -136,3 +136,28 @@ def get_aggregate_data(self, alert_id):
             :class:`py42.response.Py42Response`
         """
         return self._alert_service.get_aggregate_data(alert_id)
+
+    def get_all_alert_details(self, query):
+        """
+        Helper method that combines :func:`.search_all_pages()` and :func:`.get_details()`.
```

Comment:
The main reason to use this method is to get access to the alert observations, I suppose?
Should we mention that at all, that if you need the observations of all alerts, then use this?  Just an idea.

Reply:
yeah, that's a good clarification of what "details" actually means.